MAUI/ImageEditor/migration.md

```diff
@@ -0,0 +1,517 @@
+<table>
+<tr>
+<th>Xamarin SfImageEditor</th>
+<th>.NET MAUI SfImageEditor</th>
+<th>Description</th></tr>
+<tr>
+<td>Angle</td>
+<td>RotationAngle</td>
+<td>Gets or sets the initial rotation angle for text.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>TextStyle</td>
+<td>Gets or sets the text style for text.</td>
+</tr>
+
+<tr>
+<td>RotatableElement</td>
+<td>IsRotatable</td>
+<td>Gets or sets a value indicating whether the text is rotatable or not.</td>
+</tr>
+
+<tr>
+<td>TextAlignment</td>
+<td>TextAlignment</td>
+<td>Gets or sets the TextAlignment for the text.</td>
+</tr>
+
+<tr>
+<td>IsEditable</td>
+<td>IsEditable</td>
+<td>Gets or sets a value indicating whether the text is editable or not.</td>
+</tr>
+</table>
+
+#### ImageEditorTextStyle
+
+<table>
+<tr>
+<th>Xamarin SfImageEditor</th>
+<th>.NET MAUI SfImageEditor</th>
+<th>Description</th></tr>
+<tr>
+<td>Color</td>
+<td>TextColor</td>
+<td>Gets or sets the color for the text.</td>
+</tr>
+
+<tr>
+<td>FontSize</td>
+<td>FontSize</td>
+<td>Gets or sets the font size of the text.</td>
+</tr>
+
+<tr>
+<td>FontFamily</td>
+<td>FontFamily</td>
+<td>Gets or sets the font family for the text.</td>
+</tr>
+
+<tr>
+<td>TextEffects</td>
+<td>FontAttribute</td>
+<td>Gets or sets the font attribute for the text.</td>
+</tr>
+</table>
+
+## Enums
+
+<table>
+<tr>
+<th>Xamarin SfImageEditor</th>
+<th>.NET MAUI SfImageEditor</th>
+<th>Description</th>
+</tr>
+
+<tr>
+<td>ShapeType</td>
+<td>AnnotationShape</td>
+<td>Specifies the annotation shapes in image editor.</td>
+</tr>
+
+<tr>
+<td>FlipDirection</td>
+<td>ImageFlipDirection</td>
+<td>Specifies the direction to flip an image.</td>
+</tr>
+
+<tr>
+<td>ImageEffect</td>
+<td>ImageEffect</td>
+<td>Specifies the image effect.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>ImageFileType</td>
+<td>Specifies the image file type.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>ImageCropType</td>
+<td>Specifies the image crop types.</td>
+</tr>
+</table>
+
+## Events
+
+<table>
+<tr>
+<th>Xamarin SfImageEditor</th>
+<th>.NET MAUI SfImageEditor</th>
+<th>Description</th></tr>
+
+<tr>
+<td>BeginReset</td>
+<td>BeginReset</td>
+<td>Occurs when the image reset action.</td>
+</tr>
+
+<tr>
+<td>ImageSaving</td>
+<td>ImageSaving</td>
+<td>Occurs when the image saving.</td>
+</tr>
+
+<tr>
+<td>ImageSaved</td>
+<td>ImageSaved</td>
+<td>Occurs when the image saved in local storage.</td>
+</tr>
+
+<tr>
+<td>ItemSelected</td>
+<td>AnnotationSelected</td>
+<td>Occurs when an annotation view selected.</td>
+</tr>
+
+<tr>
+<td>ImageLoaded</td>
+<td>ImageLoaded</td>
+<td>Occurs when image loaded into view initially.</td>
+</tr>
+</table>
+
+## Methods
+
+<table>
+<tr>
+<th>Xamarin SfImageEditor</th>
+<th>.NET MAUI SfImageEditor</th>
+<th>Description</th></tr>
+
+<tr>
+<td>Rotate</td>
+<td>Rotate</td>
+<td>Method to rotate image 90 degrees in clockwise direction.</td>
+</tr>
+
+<tr>
+<td>Flip</td>
+<td>Flip</td>
+<td>Method to reverses the image along its horizontal or vertical axis.</td>

```

Comment:
<td>Method to reverse the image along its horizontal or vertical axis.</td>

Reply:
content updated